src/containers/Widgets/Widgets.js

```diff
@@ -28,7 +28,10 @@ export default class Widgets extends Component {
   static reduxAsyncConnect(params, store) {
     const {dispatch, getState} = store;
     if (!isLoaded(getState())) {
-      return dispatch(loadWidgets());
+      const promise = dispatch(loadWidgets());
+      if (__SERVER__) {
```

Comment:
> ... depends where we are - on server or on client...

check window object may work in this case

```
const isServerSide = typeof window !== 'object';
```

reference PR at redux-async-connect [#23](https://github.com/Rezonans/redux-async-connect/pull/23)